contributed_definitions/nyaml/NXfit.yaml

```
@@ -0,0 +1,263 @@
+      dimensions:
+        dim: [[1, n_points]]
+    input_independent(NX_NUMBER):
+      unit: NX_ANY
+      doc: |
+        Independent axis for this fit procedure.
+      dimensions:
+        dim: [[1, n_independent], [2, n_points]]
+    envelope(NX_NUMBER):
+      unit: NX_ANY
+      doc: |
+        Resulting envelope of all NXpeak instances.
+        Should have the same dimension as the `position` axes of the individual peaks.
+      dimensions:
+        dim: [[1, n_independent], [2, n_points]]
+    residual(NX_NUMBER):
+      unit: NX_ANY
+      doc: |
+        Difference between the envelope and the data to be fitted.
+      dimensions:
+        dim: [[1, n_independent], [2, n_points]]
+  peakPEAK(NXpeak):
+    doc: |
+      One spectral peak of the peak model.
+      It is envisioned that peaks are labeled as peak_0, peak_1, and so on.
+    total_area(NX_NUMBER):
+      unit: NX_ANY
+      doc: |
+        Total area under the curve (can also be used for the total area minus any
+        background values).
+    relative_sensitivity_factor(NX_NUMBER):
+      unit: NX_UNITLESS
+      doc: |
+        Relative sensitivity for this peak, to be used for quantification in
+        an NXprocess.
+        
+        As an example, in X-ray spectroscopy could depend on the energy scale
+        (see position), the ionization cross section, and the element probed.
+    relative_area(NX_NUMBER):
+      unit: NX_ANY
+      doc: |
+        Relative area of this peak compared to other peaks.
+        
+        The relative area can simply be derived by dividing the total_area by the
+        total area of all peaks or by a more complicated method (e.g., by
+        additionally dividing by the relative sensitivity factors). Details shall
+        be given in fitting_method.
+  backgroundBACKGROUND(NXfit_background):
+    doc: |
+      One fitted background (functional, position, and intensities) of the peak fit.
+      It is envisioned that peaks are labeled as background_0, background_1, and
+      so on.
+  fit_function(NXfit_function):
+    doc: |
+      Function used to optimize the parameters during peak fitting.
+    description:
+      doc: |
+        Description of the method used to optimize the parameters during peak fitting.
+        Examples:
+        - least squares
+        - non-linear least squares
+        - Levenberg-Marquardt algorithm (damped least-squares)
+        - linear regression
+        - Bayesian linear regression
+  figure_of_meritMETRIC(NX_NUMBER):
+    unit: NX_UNITLESS
+    doc: |
+      Figure-of-merit to determine the goodness of fit, i.e., how well the peak model
+      fits the measured observations.
+      
+      This value (which is a single number) is oftenused to guide adjustments to the
+      fitting parameters in the peak fitting process.
+    \@metric:
+      doc: |
+        Metric used to determine the goodness of fit. Examples include:
+        - :math:`\chi^2`, the standard deviation of the residuals
```

The standard deviation of residuals would correspond more to the reduced chi2 (even this not, as it's normalized to the reduced DOF)
```suggestion
        - :math:`\chi^2`, the squared sum of the sigma-weighted residuals
```

---

ok, fixed